painter.go

```diff
@@ -40,7 +40,7 @@ type PainterOptions struct {
 
 type PainterOption func(*Painter)
 
-type TicksOption struct {
+type ticksOption struct {
```

Comment:
TODO: let's make these and `multiTextOption` field names lower case for consistency